rclcpp/test/rclcpp/executors/test_executors.cpp

```diff
@@ -526,3 +526,72 @@ TYPED_TEST(TestExecutorsStable, spinSome) {
 
   spinner.join();
 }
+
+// Check spin_node_until_future_complete with node base pointer
+TYPED_TEST(TestExecutorsStable, testSpinNodeUntilFutureCompleteNodeBasePtr) {
+  using ExecutorType = TypeParam;
```

Comment:
Can I ask what's the purpose of that using? wouldn't it be the same use `TypeParam` directly in the `executor` declaration?

Reply:
I'm honestly not sure.  I just copied it from the examples above.

Reply:
I added that using back when I wrote it because I liked being more explicit about what TypeParam actually was. 

```
TypeParam executor;
```
Seemed less descriptive